pkg/scalers/redis_scaler.go

```diff
@@ -46,31 +47,36 @@ type redisScaler struct {
 }
 
 type redisConnectionInfo struct {
-	addresses        []string
-	username         string
-	password         string
-	sentinelUsername string
-	sentinelPassword string
-	sentinelMaster   string
-	hosts            []string
-	ports            []string
-	enableTLS        bool
-	unsafeSsl        bool
-	cert             string
-	key              string
-	keyPassword      string
-	ca               string
+	Addresses        []string `keda:"name=address;addresses,       order=triggerMetadata;authParams;resolvedEnv"`
+	Username         string   `keda:"name=username,       order=triggerMetadata;resolvedEnv;authParams"`
+	Password         string   `keda:"name=password,       order=triggerMetadata;resolvedEnv;authParams"`
+	SentinelUsername string   `keda:"name=sentinelUsername,       order=triggerMetadata;authParams;resolvedEnv"`
+	SentinelPassword string   `keda:"name=sentinelPassword,       order=triggerMetadata;authParams;resolvedEnv"`
+	SentinelMaster   string   `keda:"name=sentinelMaster,       order=triggerMetadata;authParams;resolvedEnv"`
+	Hosts            []string `keda:"name=host;hosts,       order=triggerMetadata;resolvedEnv;authParams"`
+	Ports            []string `keda:"name=port;ports,       order=triggerMetadata;resolvedEnv;authParams"`
+	EnableTLS        bool     `keda:"name=enableTLS;tls,       order=triggerMetadata;authParams, optional, default=false"`
+	UnsafeSsl        bool     `keda:"name=unsafeSsl,       order=triggerMetadata, optional, default=false"`
+	Cert             string   `keda:"name=Cert;cert,       order=authParams"`
```

Comment:
Similar question there ? Are you trying to cover all cases where user can mistakenly input a value into the manifest or such cases have been supported in the current code ? :thinking:

Reply:
I think multi-name keys is a good feature, although I wasn't able to find `Cert` (with capital `C`) anywhere in the original code. Maybe this is a typo?